src/tests/ftest/launch.py

```diff
@@ -196,22 +196,19 @@ def set_test_environment(args):
     print("Before Environment:")
     for name in ("OFI_INTERFACE", "OFI_DOMAIN", "CRT_PHY_ADDR_STR"):
         print(" {0:>20}: {1}".format(name, os.environ.get(name, "No env")))
+    fi_info_path = os.path.join(os.path.sep, base_dir, bin_dir, "fi_info")
 
     print("fi_info : ")
-    out = get_output("command -v fi_info")
-    print(out)
-
-    print("fi_info : ")
-    out = get_output("fi_info")
+    out = get_output("command -v {}".format(fi_info_path))
```

Comment:
I'm not sure this makes a lot of sense with a path already prefixed to it.  `command -v` is usually used to find where in PATH a command exists.

Reply:
Oh. yes. you mentioned this before (which vs. command). I will change it running just the fi_info command.

Reply:
Hmm... then why did I get a failure. fi_info command not found error?
[2020-05-21T12:57:51.689Z] fi_info : 
[2020-05-21T12:57:51.689Z] Running c o m m a n d   - v   f i _ i n f o
[2020-05-21T12:57:51.689Z] Traceback (most recent call last):
[2020-05-21T12:57:51.689Z]   File "./launch.py", line 1367, in <module>
[2020-05-21T12:57:51.689Z]     main()
[2020-05-21T12:57:51.689Z]   File "./launch.py", line 1314, in main
[2020-05-21T12:57:51.689Z]     set_test_environment(args)
[2020-05-21T12:57:51.689Z]   File "./launch.py", line 201, in set_test_environment
[2020-05-21T12:57:51.689Z]     out = get_output("command -v fi_info")
[2020-05-21T12:57:51.689Z]   File "./launch.py", line 288, in get_output
[2020-05-21T12:57:51.689Z]     cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
[2020-05-21T12:57:51.689Z]   File "/usr/lib64/python2.7/subprocess.py", line 711, in _init_
[2020-05-21T12:57:51.689Z]     errread, errwrite)
[2020-05-21T12:57:51.689Z]   File "/usr/lib64/python2.7/subprocess.py", line 1327, in _execute_child
[2020-05-21T12:57:51.689Z]     raise child_exception
[2020-05-21T12:57:51.689Z] OSError: [Errno 2] No such file or directory

Reply:
I don't know, but:

```
[2020-05-21T12:57:51.689Z] Running c o m m a n d - v f i _ i n f o
```

Doesn't have any path on it, and your previous patch was trying to put a path on it so something must be going wrong there.

Reply:
One thing to note is that `get_output()` takes a list as it's first argument, not a string:
https://github.com/daos-stack/daos/blob/c4cd5e0403d23935b87ff1372fa6e1e919a9e3ab/src/tests/ftest/launch.py#L267-L271